data/org.freedesktop.portal.Account.xml

```diff
@@ -37,6 +39,14 @@
         Supported keys in the @options vardict include:
         <variablelist>
           <varlistentry>
+            <term>token s</term>
```

Comment:
I think a reference to @options in the description of GetUserInformation is missing between line 34 and line 35